src/webgpu/api/validation/texture/destroy.spec.ts

```diff
@@ -43,6 +43,8 @@ g.test('invalid_texture')
     invalidTexture.destroy();
   });
 
+const kColorTextureFormat: GPUTextureFormat = 'rgba32float';
```

Comment:
does this need to be `rgba32float`? Will `rgba8unorm` work just as well for this test?

Reply:
It also uses storage "read-write" usage. So rgba8unorm won't work without test modification.

Reply:
I'm not seeing this "read-write" usage. Am I missing it? To put it another way, I'm not seeing what breaks if this changes to `rgba8unorm`. All it tries to do is render to a destroyed texture. Any renderable texture format should cover that right?